tests/run_mapserver.sh

```
@@ -8,11 +8,11 @@ then
 fi
 
 for fn in $(find . -type f -name '*.map' -print); do
```

Shellcheck (0.11.0)

[warning] For loops over find output are fragile. Use find -exec or a while read loop.

(SC2044)